test/functional/test-layout.js

```diff
@@ -400,7 +400,7 @@ describe('Layout', () => {
   it('layout=fluid - default', () => {
     div.setAttribute('height', 'fluid');
     const parentDiv = document.createElement('div');
-    div.parentElement = parentDiv;
+    Object.defineProperty(div, 'parentElement', {value: parentDiv});
```

Comment:
I think this can just be dropped. The `appendChild` below will set `parentElement`

Reply:
Done.